### ISSUE TITLE
Value with <> Angle brackets are not being displayed.

### ISSUE THREAD

**Issue by Norlandz** [mwouts/itables#346]
eg:

![image](https://github.com/user-attachments/assets/b8121be5-c010-42da-a4ee-6f493cc0b270)

```
import pandas as pd

from itables import init_notebook_mode, show
init_notebook_mode(all_interactive=True, connected=False)

df = pd.DataFrame(
    [
        ["aa", "bb"],
        ["v11", "v12"],
        ["v21", "v22<bl>extra"],
    ]
)
df
```

I guess the parsing of string treated the `<bl>` as an html tag. But it shouldnt.

**Comment by mwouts** [comment]
Good point! I have added a new `allow_html=False` option to ITables v2.4.0.

Could you give a try at the release candidate https://pypi.org/project/itables/2.4.0rc0/ (`pip install itables --pre`), and let me know whether it fixes the above? Thanks!

**Comment by Norlandz** [comment]
@mwouts (thanks for the update. Im bit busy right now, currently not on any python project, maybe a month later)

**Comment by mattkeanny** [comment]
Great library, used datatables in the past and glad it has been made usable from notebooks.
Tested the `allow_html=False` option in the 2.4.0rc0 version, works as expected. 
Thanks.

**Comment by mwouts** [comment]
Thanks for testing! The new release is now available on pip: `pip install itables==2.4.0`. Thanks @Norlandz and @mattkeanny for reporting the issue and testing the release candidate !
